src/histolab/tile.py

```diff
@@ -0,0 +1,161 @@
+import os
+from pathlib import Path
+
+import numpy as np
+from PIL import Image
+
+from .filters import image_filters as imf
+from .filters import morphological_filters as mof
+from .filters.util import mask_percent
+from .types import CoordinatePair
+
+
+class Tile:
+    def __init__(self, image: Image.Image, coords: CoordinatePair, level: int = 0):
+        self._image = image
+        self._level = level
+        self._coords = coords
+
+    @property
+    def image(self) -> Image.Image:
+        return self._image
+
+    @property
+    def level(self) -> int:
+        return self._level
+
+    @property
+    def coords(self) -> CoordinatePair:
+        return self._coords
+
+    def has_enough_tissue(
+        self, tissue_percent: float = 80.0, near_zero_var_threshold: float = 0.1
+    ) -> bool:
+        """Check if the tile has enough tissue.
+
+        Parameters
+        ----------
+        tissue_percent : float, optional
+            Number between 0.0 and 100.0 representing the minimum required percentage of
+            tissue over the total area of the image, default is 80.0
+        near_zero_var_threshold : float, optional
+            Minimum image variance after morphological operations (dilation, fill
+            holes), default is 0.1
+
+        Returns
+        -------
+        enough_tissue : bool
+            Whether the image has enough tissue, i.e. if the proportion of tissue
+            over the total area of the image is more than ``threshold`` and the image
+            variance after morphological operations is more than
+            ``near_zero_var_threshold``.
+        """
+
+        if self._is_almost_white:
+            return False
+
+        if self._is_all_tissue_or_no_tissue(near_zero_var_threshold):
```

Comment:
I do not remember why "all tissue" is a problem

Reply:
It's not actually a problem. The point is: if we check for "no tissue" using the variance of the image, "no tissue" is the same as "all tissue". We probably need to find another way to determine if we have no tissue in the tile, then.

Reply:
IMHO this prop name is odd, considering that it returns a bool, can we call it just `_is_all_tissue` ?

Reply:
The point is that this evaluates to True both for cases of ALL tissue and cases of NO tissue at all. Mmmmm

Reply:
yes but it's a bool and the name can be more simple than that, can we try to change that name? @nicolebussola what do you think about it?